modules/ossm-upgrade-22-23-changes.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+:_content-type: CONCEPT
+[id="ossm-upgrade-22-23-changes_{context}"]
+= Upgrade changes from version 2.2 to version 2.3
+
+Upgrading the {SMProductShortName} control plane from version 2.2 to 2.3 introduces the following behavioral changes:
+
+* This release requires use of `WasmPlugin` API. Support for `ServiceMeshExtension` API, which was deprecated in 2.2, has now been removed. If you attempt to upgrade while using `ServiceMeshExtension` API, the upgrade fails.
+
+* The Elasticsearch Operator has been deprecated and OpenShift 4.11 is the last version to support it.
```

Comment:
I don't think this belongs to this PR. OSSM releases are independent of OCP releases. OSSM 2.3 will also work on OCP 4.10, right?

Reply:
Yes, 2.3 work on OCP 4.10 also

Reply:
@jwendell, thanks for expressing you find this confusing. It helps to get perspective from different people. 

This topic is intended to serve a different function than release notes. Here, we're trying to emphasize what's different in the upgrade process closer to where core docs has instructions for upgrading the control plane. The release notes is a separate document, so if we don't communicate this information close to the instructions, then the customer has to look in two different documents. In that sense, I think telling customers what might cause the upgrade to fail is beneficial.

On the second bullet, I think I agree with you. Looking at this again, the information about elasticsearch operator seems like it belongs in the release notes rather than here. 

Does that make sense?